src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -7,190 +7,190 @@ partial class ApplicationEngine
-            [OpCode.LDLOC] = 60,
-            [OpCode.STLOC0] = 60,
-            [OpCode.STLOC1] = 60,
-            [OpCode.STLOC2] = 60,
-            [OpCode.STLOC3] = 60,
-            [OpCode.STLOC4] = 60,
-            [OpCode.STLOC5] = 60,
-            [OpCode.STLOC6] = 60,
-            [OpCode.STLOC] = 60,
-            [OpCode.LDARG0] = 60,
-            [OpCode.LDARG1] = 60,
-            [OpCode.LDARG2] = 60,
-            [OpCode.LDARG3] = 60,
-            [OpCode.LDARG4] = 60,
-            [OpCode.LDARG5] = 60,
-            [OpCode.LDARG6] = 60,
-            [OpCode.LDARG] = 60,
-            [OpCode.STARG0] = 60,
-            [OpCode.STARG1] = 60,
-            [OpCode.STARG2] = 60,
-            [OpCode.STARG3] = 60,
-            [OpCode.STARG4] = 60,
-            [OpCode.STARG5] = 60,
-            [OpCode.STARG6] = 60,
-            [OpCode.STARG] = 60,
-            [OpCode.NEWBUFFER] = 80000,
-            [OpCode.MEMCPY] = 80000,
-            [OpCode.CAT] = 80000,
-            [OpCode.SUBSTR] = 80000,
-            [OpCode.LEFT] = 80000,
-            [OpCode.RIGHT] = 80000,
-            [OpCode.INVERT] = 100,
-            [OpCode.AND] = 200,
-            [OpCode.OR] = 200,
-            [OpCode.XOR] = 200,
-            [OpCode.EQUAL] = 1000,
-            [OpCode.NOTEQUAL] = 1000,
-            [OpCode.SIGN] = 100,
-            [OpCode.ABS] = 100,
-            [OpCode.NEGATE] = 100,
-            [OpCode.INC] = 100,
-            [OpCode.DEC] = 100,
-            [OpCode.ADD] = 200,
-            [OpCode.SUB] = 200,
-            [OpCode.MUL] = 300,
-            [OpCode.DIV] = 300,
-            [OpCode.MOD] = 300,
-            [OpCode.SHL] = 300,
-            [OpCode.SHR] = 300,
-            [OpCode.NOT] = 100,
-            [OpCode.BOOLAND] = 200,
-            [OpCode.BOOLOR] = 200,
-            [OpCode.NZ] = 100,
-            [OpCode.NUMEQUAL] = 200,
-            [OpCode.NUMNOTEQUAL] = 200,
-            [OpCode.LT] = 200,
-            [OpCode.LE] = 200,
-            [OpCode.GT] = 200,
-            [OpCode.GE] = 200,
-            [OpCode.MIN] = 200,
-            [OpCode.MAX] = 200,
-            [OpCode.WITHIN] = 200,
-            [OpCode.PACK] = 15000,
-            [OpCode.UNPACK] = 15000,
-            [OpCode.NEWARRAY0] = 400,
-            [OpCode.NEWARRAY] = 15000,
-            [OpCode.NEWARRAY_T] = 15000,
-            [OpCode.NEWSTRUCT0] = 400,
-            [OpCode.NEWSTRUCT] = 15000,
-            [OpCode.NEWMAP] = 200,
-            [OpCode.SIZE] = 150,
-            [OpCode.HASKEY] = 270000,
-            [OpCode.KEYS] = 500,
-            [OpCode.VALUES] = 270000,
-            [OpCode.PICKITEM] = 270000,
-            [OpCode.APPEND] = 270000,
-            [OpCode.SETITEM] = 270000,
-            [OpCode.REVERSEITEMS] = 270000,
-            [OpCode.REMOVE] = 500,
-            [OpCode.CLEARITEMS] = 400,
-            [OpCode.ISNULL] = 60,
-            [OpCode.ISTYPE] = 60,
-            [OpCode.CONVERT] = 80000,
+            [OpCode.DEPTH] = 1 << 1,
```

Comment:
It's similar to PUSHINT, so if we're to change them, it needs to be changed also.